package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/nmea0183-signalk",
-  "version": "3.6.0",
+  "version": "3.6.1",
```

Comment:
Please don't update the version number since you don't know which version this would go into. That will be done when we publish a new version.